packages/plugins/src/cli/setup.ts

```diff
@@ -137,31 +137,23 @@ export const makeCLI = (
       'install',
       'Installs the required version of the Sphinx Solidity library contracts and Sphinx Foundry fork',
       (y) =>
-        y
-          .usage('Usage: sphinx install')
-          .option('skip-library', {
-            describe:
-              'Skips installing the Sphinx library and only installs the Sphinx Foundry fork.',
-            boolean: true,
-            default: false,
-          })
-          .option('ci', {
-            describe:
-              'Continuous integration mode (skips installing the Sphinx Foundry fork)',
-            boolean: true,
-            default: false,
-          }),
+        y.usage('Usage: sphinx install').option('ci', {
+          describe:
+            'Continuous integration mode (skips installing the Sphinx Foundry fork)',
```

Comment:
I think we should remove the `ci` flag description since it's deprecated. If you disagree, I think the description should at least be updated so that it doesn't mention the Foundry fork